test/v8-profiler.js

```diff
@@ -1,7 +1,18 @@
-const expect  = require('chai').expect,
+"use strict";
+const expect = require('chai').expect,
+      path = require('path'),
       profiler = require('../');
 
 const NODE_V_010 = /^v0\.10\.\d+$/.test(process.version);
+const SOURCE_PATH = path.join(__dirname, '..');
+
+var workerThreads;
```

Comment:
Here could be `let`

Reply:
Breaks Node 4 and 5 on Travis

Reply:
Actually might just need a "use strict" in this file.

Reply:
> Actually might just need a "use strict" in this file.

enen